android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -90,7 +90,11 @@
     }
 
     function getOskHeight() {
-      return oskHeight;
+      var height = oskHeight;
+      if(keyman.osk.banner._activeType != 'blank') {
+        height = height; // Android's perspective - banner height is wholly separate.
+      }
```

Comment:
I was going to have this match how iOS manages things - it changes the set height based on if the banner's displayed.  But... the Android paradigm is different, so I had to roll it back.

Now, just to revert the change while adding documentation for why it's different.

Reply:
Oh, of course.  That note was a reminder-to-self bit; these comments were end-of-day.